src/lib.rs

```diff
@@ -497,10 +530,18 @@ pub trait Handler {
         Err(Error::Unimplemented)
     }
 
+    /// Return the general registers.  The registers are returned as a
+    /// vector of bytes, with the registers appearing back-to-back in
+    /// a target-specific order, with the bytes laid out in the target
+    /// byte order.
```

Comment:
I'm not super-keen on "back-to-back", but I don't know what a better term is here.  "Contiguous"?